benchmark/suite/macro/lib/csv-set.rkt

```diff
@@ -0,0 +1,106 @@
+#lang racket
```

Comment:
there should be a header comment explaining what is the purpose of this file

Reply:
I agree.
It will be something like:
This program allows you to compare two files of floating point numbers for agreement up to  a given delta.

Reply:
Done.